data/coc.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: default
+title: Code Of Conduct
+header: Code Of Conduct
+bloglike: true
+nav: code-of-conduct
+---
+
+## The Short Form
+
+We invite you to come to PyCon India 2023 in a spirit of curiosity,
+friendliness, open-mindedness, and respect. We will not tolerate
+harassment in any form.
+
+At the PyCon India venue and related online forums, you agree to:
+
+- Be kind and sensitive to the people around you and respect the
+    boundaries of other attendees.
+
+- Attempt to resolve difference of opinions peacefully
+
+- Avoid harassment, exclusionary or offensive behaviour of any
+    kind. This includes but is not limited to offensive verbal
+    comments related to gender, sexual orientation, disability,
+    physical appearance, body size, race, religion, sexual images in
+    public spaces, deliberate intimidation, stalking, following,
+    harassing photography or recording, sustained disruption of talks
+    or other events, inappropriate physical contact, and unwelcome
+    sexual attention.
+
+Violations can be reported online by sending a mail to
+[coc@in.pycon.org](mailto:coc@in.pycon.org), or to PyCon India
+volunteers at the venue.
+
+Violations of the Code of Conduct can result in expulsion from the
+conference with no refund. Please read on for a more detailed version
+of the Code of Conduct below.
+
+## The Long Form
+
+### Aim
+
+PyCon India is a community organized conference intended for
+advocating the use and adoption of the Python programming language in
+India. It is also a platform for fostering networking and
+collaboration among the Python developer community in India.
+
+We believe in the broad spectrum of individual and collective
+experience and in the inherent dignity of all people. We believe that
+amazing things happen when people from different worlds and
+world-views approach each other to create a conversation.
+
+PyCon India 2023 welcomes everyone and is dedicated to providing a
+harassment-free conference experience for everyone, regardless of age,
+gender, sexual orientation, physical appearance, disability,
+neurotype, race, religion, political opinion, culture or
+employment. We don’t tolerate harassment of attendees in any form.
+
+### Where
+
+This Code of Conduct applies to the venue of PyCon India 2023
+Conference, pre-event & post-event informal gatherings, related online
+forums and all conference related events organized by PyCon India.
+
+### Who
+
+This is applicable to speakers, participants, sponsors, volunteers and
+organizers.
+
+### Be The Best You Can Be
+
+- Be kind, sensitive, understanding and respectful to other
+    attendees.
+
+- Listen with empathy when someone has a different perspective.
+  
+- Talk to the volunteers if you notice that something could be
+    better.
```

Comment:
I don't think the CoC is the right place to encourage people to "share feedback"

We should instead do something on the lines of"Talk to the volunteers if you see something inappropriate or feel something is wrong"

Reply:
Agree, let me re-phrase based on what you shared.